src/Orleans.Runtime/Hosting/CoreHostingExtensions.cs

```diff
@@ -95,11 +110,62 @@ public static ISiloHostBuilder ConfigureSiloName(this ISiloHostBuilder builder,
         }
 
         /// <summary>
-        /// Configure silo to use Development membership
+        /// Configures a localhost silo for development and testing.
         /// </summary>
-        public static ISiloHostBuilder UseDevelopmentClustering(this ISiloHostBuilder builder, Action<DevelopmentMembershipOptions> configureOptions)
+        /// <param name="builder">The silo builder.</param>
+        /// <param name="siloPort">The silo port.</param>
+        /// <param name="gatewayPort">The gateway port.</param>
+        /// <param name="primarySiloEndpoint">
+        /// The endpoint of the primary silo, or <see langword="null"/> to use this silo as the primary.
+        /// </param>
+        /// <returns>The silo builder.</returns>
+        public static ISiloHostBuilder UseLocalhostClustering(
+            this ISiloHostBuilder builder,
+            int siloPort = EndpointOptions.DEFAULT_SILO_PORT,
+            int gatewayPort = EndpointOptions.DEFAULT_GATEWAY_PORT,
+            IPEndPoint primarySiloEndpoint = null,
```

Comment:
NIT: can we use its real default here, instead of null? Can make code more cleaner

Reply:
We can't, only compile-time constants can be used as defaults